src/Components/PlaceList.js

```diff
@@ -0,0 +1,126 @@
+import { Link } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { fetchPlaces } from '../Redux/places/placesSlice';
+import { fetchDetailsPage } from '../Redux/places/detailsPageSlice';
+import '../Placelist.css';
+
+function PlacesList() {
```

Comment:
- Please, consider using arrow funcs according to Javascript ES6 good practices.